debug-bar.php

```diff
@@ -125,6 +125,10 @@ function enqueue() {
 
 		wp_enqueue_script( 'debug-bar', plugins_url( "js/debug-bar$suffix.js", __FILE__ ), array( 'jquery' ), '20170623', true );
 
+		if ( defined( 'SAVEQUERIES' ) && SAVEQUERIES || ! empty( $GLOBALS['EZSQL_ERROR'] ) ) {
+			wp_enqueue_script( 'debug-bar-queries', plugins_url( "js/debug-bar-queries$suffix.js", __FILE__ ), array( 'jquery' ), '20170808' );
```

Comment:
Does this script need to be loaded in the header ? or can it be loaded after the page has rendered ?
I'd suggest setting `$in_footer` to `true`.
https://developer.wordpress.org/reference/functions/wp_enqueue_script/

Reply:
can definitely go in footer.

Reply:
can definitely go in the footer.